### ISSUE TITLE
It looks like the drop-down list of topics may be getting too big 

### ISSUE THREAD

**Issue by neilideal** [sk22/megalodon#283]
**Is your feature request related to a problem? Please describe.**
It looks like the drop-down list of topics may be getting too big. After reading issue [#271](https://github.com/sk22/megalodon/issues/271), I decided to look in the settings and see the current themes. On my 1080*2400 screen, it was already taking up half the height and I don't think it's very pretty. Suggestions to add a new topic are likely to keep coming up. So, at some point, the list may become so large that it won't fit on one screen. And I think this will become a problem.

**Describe the solution you'd like**
As a result, I decided to take a screenshot of the settings screen and make the simplest possible sketch of the new screen to select the main color of the application. I tried to follow the guidelines, but I couldn't repeat them completely, so please don't judge me harshly.

![Select_theme_color_page](https://user-images.githubusercontent.com/62957728/212120147-64f0055a-f53d-4fab-950a-d116e97e7b9c.png)
![Idea](https://user-images.githubusercontent.com/62957728/212120533-bbd21ae3-f353-4434-834b-0b59ea104550.png)

**Comment by neilideal** [comment]
Also, @LucasGGamerM, it's your problem in Moshidon too

**Comment by neilideal** [comment]
In general, I now think it would be a good idea to take out all the appearance settings and implement a preview of the colors in the form of a pre-prepared image of the feed. Naturally, in three modes (light, dark, true black). The user could use horizontal scrolling to choose his color. The choice could be marked as in the case of selecting the theme mode. I'm going to try to do that, but a little later.

**Comment by sk22** [comment]
good idea, thanks! but i think the color list will be enough, like in your screenshot. i really don't want to implement images for every theme ^^'

**Comment by LucasGGamerM** [comment]
Implementing images for every theme would be painful, but I like the drop-down for now
